protostar-rust/src/main.rs

```diff
@@ -0,0 +1,107 @@
+use anyhow::{anyhow, Context, Result};
+use cairo_lang_protostar::test_collector::LinkedLibrary;
+use clap::Parser;
+use rust_test_runner_bindings::run_tests;
+use scarb_metadata::{Metadata, MetadataCommand, PackageId, PackageMetadata};
+use serde::Deserialize;
+use std::env::set_var;
+use std::path::PathBuf;
+
+const CONTRACT_TARGET: &str = "starknet-contract";
+
+#[derive(Parser, Debug)]
+struct Args {
+    // #[clap(required = true)]
+    // paths: Vec<PathBuf>,
```

Comment:
```suggestion
```

Reply:
Don't we need to provide it?

Reply:
No. The way `protostar test` will now work is similar to `cargo test`: By default it will run all tests within the workspace. 

Scarb metadata is already aware of current working directory and detects scarb workspaces within it, so I just run tests on the paths that it provides .

Reply:
I've updated the args to remove unnecessary code. They will be expanded in later PRs.